app/src/main/java/org/wikipedia/util/GeoUtil.kt

```diff
@@ -51,4 +51,10 @@ object GeoUtil {
             "${formatter.format(distance)} km"
         }
     }
+
+    fun isSamePlace(startLat: Double, endLat: Double, startLon: Double, endLon: Double): Boolean {
+        val df = DecimalFormat("#.#######")
```

Comment:
Instead of comparing string versions of `double` variables, can we just compare the `double`s? (within a tolerance of 0.0000001)?

Reply:
Done!